.github/workflows/push.yml

```diff
@@ -58,7 +58,9 @@ jobs:
           python-version: '3.9'
 
       - name: Install uv
-        uses: astral-sh/setup-uv@887a942a15af3a7626099df99e897a18d9e5ab3a # v5.1.0
+        uses: astral-sh/setup-uv@f94ec6bedd8674c4426838e6b50417d36b6ab231 # v5.3.1
+        with:
+          version: "0.6.5"
```

Comment:
> Update uv to 0.6.5

What you do here is pinning, not upgrading, correct? We were already on latest version, since we did not specify one.

AFAIK we did not have any problems with being on latest.

Reply:
The problem is that we are many versions behind the latest. We can either say that we are always using the latest or pin version, both will work for me. The reason why I upgraded in the first place is that CI was failing for the code that worked locally.